src/nautilus_librarian/typer/commands/workflows/actions/validate_images_dimensions_action.py

```diff
@@ -0,0 +1,42 @@
+from nautilus_librarian.mods.dvc.domain.utils import (
+    extract_list_of_media_file_changes_from_dvc_diff_output,
+)
+from nautilus_librarian.mods.libvips.domain.validate_image_dimensions import (
+    validate_image_dimensions,
+)
+from nautilus_librarian.typer.commands.workflows.actions.action_result import (
+    ActionResult,
+    ErrorMessage,
+    Message,
+    ResultCode,
+)
+
+
+def validate_images_dimensions(dvc_diff, min_image_size, max_image_size):
+    """
+    It validates all the media sizes in the dvc diff.
+    """
+    if dvc_diff == "{}":
+        return ActionResult(ResultCode.EXIT, [Message("No Gold image changes found")])
+
+    filenames = extract_list_of_media_file_changes_from_dvc_diff_output(
+        dvc_diff, only_basename=False
+    )
+
+    messages = []
+
+    for filename in filenames:
+        try:
+            (width, height) = validate_image_dimensions(
+                filename, min_image_size, max_image_size
+            )
+            messages.append(
+                Message(f"Dimensions of {filename} are {width} x {height} ✓")
```

Comment:
This is not related to this PR but I think we should define the format of the output we want for the application. I think we could use a format like this:

```
"✓ Dimensions of {filename} are {width} x {height}
"✗ Dimensions of {filename} are wrong ...
"✓ ok ...
"✗ error ...
```

Always put the ✓ or the ✗ at the beginning.

The same for json format if we start using it.

Reply:
I'll use first proposed alternatives for the moment, until we have a "standard"